cmd/demo/node.go

```diff
@@ -122,6 +126,17 @@ func importAccount(secret string) (*ewallet.Wallet, *ewallet.Account, error) {
 	return wallet, acc.(*ewallet.Account), err
 }
 
+func getOnChainBal(addressList []wallet.Address) (balList []*big.Int, err error) {
+	for _, addr := range addressList {
+		bal, err := ethereumBackend.BalanceAt(context.Background(), common.BytesToAddress(addr.Bytes()), nil)
+		if err != nil {
+			return []*big.Int{}, err
```

Comment:
`errors.Wrap(err, "querying on-chain balances")` then you dont need the `errors.WithStack` further down.  
Also just `nil` instead of `[]*big.Int{}` is ok.

Reply:
Addressed in this commit - 85c0766.

Reply:
Addressed in this commit - 85c0766.